### PR TITLE
enhance(controller): support baidu oss offical sdk

### PR DESCRIPTION
**Pull request by jialeicui**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class StorageAccessServiceBos implements StorageAccessService {
+
+    private final String bucket;
+
+    private final long partSize;
+
+    private final BosClient ossClient;
```
Comment (xuchuan) [inline server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java:59]: bosClient

**Review by xuchuan — CHANGES_REQUESTED** [review]

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java (hunk 1 above)
Comment (jialeicui) [reply server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java]: Done

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        var config = new BosClientConfiguration();
+        config.setCredentials(new DefaultBceCredentials(s3Config.getAccessKey(), s3Config.getSecretKey()));
+        if (s3Config.getEndpoint() != null) {
+            config.setEndpoint(s3Config.getEndpoint());
+            config.setCnameEnabled(true);
+        }
+        this.bosClient = new BosClient(config);
+    }
+
+    private boolean isNotFount(BceServiceException e) {
```
Comment (anda-ren) [inline server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java:76]: typo for Found?

**Review by anda-ren — APPROVED** [review]
LGTM

File: server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java (hunk 2 above)
Comment (jialeicui) [reply server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/baidu/StorageAccessServiceBos.java]: Done

**Review by anda-ren — APPROVED** [review]
LGTM

**Review by xuchuan — APPROVED** [review]

**Pull request merged**